connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -69,6 +71,15 @@
     private static final String QUERY_FMT_CONTACT =
             "SELECT name, account.name, MobilePhone FROM contact WHERE email = '%s' AND contact.owner.email = '%s'";
 
+    // Find all Opportunity Ids related to sender email, based on condition.
+    private static final String QUERY_FMT_CONTACT_OPPORTUNITY = "SELECT Opportunity.Id FROM OpportunityContactRole " +
```

Comment:
Per James comments on APF-1124, we should also not show any opportunities with stage name ```Closed Lost```.

https://jira-euc.eng.vmware.com/jira/browse/APF-1124?focusedCommentId=968951&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-968951

Reply:
Now its not going to include Opportunities if stage is either `Closed Lost` or `Closed Won`.